Carmunication/Controllers/MyPage/FriendListViewController.swift

```diff
@@ -198,19 +199,21 @@ extension FriendListViewController: UITableViewDataSource {
     // 셀 구성
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCell(
-            withIdentifier: "friendListTableViewCell",
+            withIdentifier: FriendListTableViewCell.cellIdentifier,
             for: indexPath
         ) as? FriendListTableViewCell else {
             return UITableViewCell()
         }
-        cell.nicknameLabel.text = friendList[indexPath.section].nickname
-        // 친구 이미지 불러오기
-        if let imageURL = friendList[indexPath.section].imageURL {
-            loadProfileImage(urlString: imageURL) { friendImage in
-                if let friendImage = friendImage {
-                    cell.profileImageView.image = friendImage
-                } else {
-                    cell.profileImageView.image = UIImage(named: "profile")
+        if let friend = friendList[indexPath.section][1] as? User {
+            cell.nicknameLabel.text = friend.nickname
+            // 친구 이미지 불러오기
+            if let imageURL = friend.imageURL {
+                loadProfileImage(urlString: imageURL) { friendImage in
+                    if let friendImage = friendImage {
+                        cell.profileImageView.image = friendImage
+                    } else {
+                        cell.profileImageView.image = UIImage(named: "profile")
+                    }
```

Comment:
이렇게 수정하면 코드가 좀 더 짧아지겠네요 ! 😄

```suggestion
                    cell.profileImageView.image = friendImage ?? UIImage(named: "profile")
```